src/paginator.ts

```diff
@@ -112,13 +110,13 @@ paginator.parseArguments_ = function(args) {
   const firstArgument = args[0];
   const lastArgument = args[args.length - 1];
 
-  if (is.fn(firstArgument)) {
+  if ((is as any).fn(firstArgument)) {
```

Comment:
As a non-TS user, could you break down what's going on here? And if there's a prettier solution, or if we should consider a different library than `is`?

Reply:
Yeah, this was super ugly.  The `@types/is` package doesn't have `is.fn` for some reason.  This will go away as soon as this lands:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/24542

Reply:
Ah, cool. Thanks!